chore: update Vaadin version to 21.0.0, revert V22 changes (#726)
vaadin/docs: articles/fusion/advanced/endpoints-generator.asciidoc

=== Modules

> The generator will collect all the `_tags_` field of all operations in the OpenAPI document.

Reviewer: ```suggestion
The generator collects all the `_tags_` field of all operations in the OpenAPI document.
```

> Each tag will generate a corresponding TypeScript file.

Reviewer: ```suggestion
Each tag generates a corresponding TypeScript file.
```

The tag name is used for TypeScript module name as well as the file name.

> TsDoc of the class will be fetched from `_description_` field of the https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#tagObject[tag object] which has the same name as the class.

Reviewer: ```suggestion
TsDoc of the class is fetched from `_description_` field of the https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#tagObject[tag object] which has the same name as the class.
```
